pkg/controller/infra/infra_controller.go

```diff
@@ -109,6 +109,7 @@ func NewInfraController(
 
 	c.syncHandler = c.syncCluster
 	c.enqueueCluster = c.enqueue
+	c.ansibleRunner = ansible.NewAnsibleRunner(c.kubeClient, c.ansibleImage, c.ansibleImagePullPolicy)
```

Comment:
Since the ansible runner is created in `NewInfraController`, the `InfraController` no longer needs to hold onto the `ansibleImage` and `ansibleImagePullPolicy`.